lib/modules/webserver/server.js

```diff
@@ -1,9 +1,18 @@
 let finalhandler = require('finalhandler');
+const fs = require('../../core/fs.js');
 let http = require('http');
 let serveStatic = require('serve-static');
 const {canonicalHost, defaultHost, dockerHostSwap} = require('../../utils/host');
+const opn = require('opn');
+const utils = require('../../utils/utils.js');
 require('http-shutdown').extend();
 
+require('ejs');
+const embark_building_placeholder = require('../code_generator/code_templates/embark-building-placeholder.html.ejs');
+
+let isFirstStart = true;
+let opened = false;
```

Comment:
Those could be object attributes of the Server class

Reply:
Done.